pkgs/hirosassa/tfcmt-gitlab/pkg.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  - name: hirosassa/tfcmt-gitlab@v0.1.4
+  - name: hirosassa/tfcmt-gitlab
+    version: v0.1.2
```

Comment:
```suggestion
```

What the reason do you add v0.1.2?
If the configuration isn't changed, this isn't needed.
And if the configuration is changed, you have to configure [version_overrides](https://aquaproj.github.io/docs/reference/registry-config/version-overrides).

Reply:
@suzuki-shunsuke Thanks for the comment. This is just a test code. I removed it in [2e85a35](https://github.com/aquaproj/aqua-registry/pull/5493/commits/2e85a35f45d515553d1f3f45e4ff3b61e6f29592)